scripts/ci-entrypoint.sh

```diff
@@ -152,6 +154,11 @@ create_cluster
 # export the target cluster KUBECONFIG if not already set
 export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
 
+# install cloud-provider-azure components, if using out-of-tree
```

Comment:
@CecileRobertMichon @lzhecheng incorporated the helm stuff into the `ci-entrypoint.sh` script itself. The way this will work is basically the same way I implemented this in the `Tiltfile` (wait for kubeconfig secret, wait for successful get nodes). Reference:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/1352fa2be39e1431f6beb62ecaa014471435b5ca/Makefile#L283-L287